pkg/apis/pxc/v1/pxc_backup_types.go

```diff
@@ -43,10 +43,10 @@ type PXCBackupState string
 
 const (
 	BackupNew       PXCBackupState = ""
-	BackupStarting                 = "Starting"
-	BackupRunning                  = "Running"
-	BackupFailed                   = "Failed"
-	BackupSucceeded                = "Succeeded"
+	BackupStarting  PXCBackupState = "Starting"
+	BackupRunning   PXCBackupState = "Running"
+	BackupFailed    PXCBackupState = "Failed"
+	BackupSucceeded PXCBackupState = "Succeeded"
 )
 
```

Comment:
Previously only BackupNew had a type of PXCBackupState. Other const had string type.

Reply:
Now all consts from this block has PXCBackupState type